.bazelrc

```diff
@@ -160,6 +160,7 @@ test:ci --nocache_test_results
 test:ci --spawn_strategy=local
 test:ci --test_output=errors
 test:ci --test_verbose_timeout_warnings
+test:air-new-persistence-mode-off --test_env=RAY_AIR_NEW_PERSISTENCE_MODE=0
```

Comment:
what is the background context? is this required for all tests?

Reply:
@matthewdeng wanted to experience the feature flag with a subset of tests, and he wants to do that for civ2 tests for some reason ;); I put it as a config in case matt wants to reuse it somewhere or change the flag, etc. it's easier for him to change in one place

Reply:
This will be needed for the core and serve tests that are failing [here](https://buildkite.com/ray-project/premerge/builds/3395)

Reply:
For the civ1 (?) tests I have been able to set the flag directly in the `bazel test` commands: https://github.com/ray-project/ray/pull/38844/files

Reply:
also why just civ2 tests? what is this trying to achieve?